src/main/java/edu/ucsb/cs156/dining/controllers/ReviewController.java

```diff
@@ -160,13 +167,16 @@ public Review editReview(@Parameter Long id, @RequestBody @Valid EditedReview in
 
         if (incoming.getReviewerComments() != null &&!incoming.getReviewerComments().trim().isEmpty()) {
             oldReview.setReviewerComments(incoming.getReviewerComments());
+            // oldReview.setStatus(ModerationStatus.AWAITING_REVIEW);
         }else{
             oldReview.setReviewerComments(null);
+            // oldReview.setStatus(ModerationStatus.APPROVED);
         }
 
```

Comment:
Commented out code does not belong in a PR targetting main, and this is not related to this PR.